contracts/mocks/extensions/tokenProperties/RMRKTokenPropertiesMock.sol

```diff
@@ -2,29 +2,25 @@
 
 pragma solidity ^0.8.16;
 
-import "./IRMRKTokenProperties.sol";
+import "../../../RMRK/extension/tokenProperties/IRMRKTokenProperties.sol";
 
 /**
  * @title RMRKTokenProperties
  * @author RMRK team
  * @notice Smart contract of the RMRK Token properties module.
  */
-contract RMRKTokenProperties is IRMRKTokenProperties {
+contract RMRKTokenPropertiesMock is IRMRKTokenProperties {
 
-    uint256 private totalStringProperties;
-    mapping(uint256 => string) stringProperties;
-    mapping(uint256 => mapping(string => uint256)) private _stringProperties;
+    mapping(string => uint256) private _keysToIds;
+    uint256 private _totalProperties = 0;
 
-    uint256 private totalAddressProperties;
-    mapping(uint256 => address) addressProperties;
-    mapping(uint256 => mapping(string => uint256)) private _addressProperties;
+    mapping(string => uint256) private _stringValuesToIds;
```

Comment:
I just realized that this was meant to be what I named `reverseDictionary`, so I'll stop adding suggestions now, but this should be accompanied by the opposite mapping as well and used in the `_stringValues`.